core/src/main/scala/org/apache/spark/errors/ExecutionErrors.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.errors
+
+import java.io.IOException
+
+import org.apache.hadoop.fs.Path
+
+import org.apache.spark.SparkException
+import org.apache.spark.storage.{BlockId, RDDBlockId}
+
+/**
+ * Object for grouping error messages from (most) exceptions thrown during query execution.
+ * This does not include exceptions thrown during the eager execution of commands, which are
+ * grouped into [[CompilationErrors]].
+ */
+private[spark] object ExecutionErrors {
```

Comment:
ditto

Reply:
Query*Errors.scala are in different module (SQL) and not callable from here (Core), so I create new class to group errors into. Is the naming reasonable or should I change them?

Reply:
@allisonwang-db We will do the job in spark core ?

Reply:
As @allisonwang-db created jira tickets for spark core, I think we will

Reply:
> @allisonwang-db We will do the job in spark core ?

Yes we will do the error grouping for all other components as well including core, mllib, etc.

Reply:
@allisonwang-db Thanks for explanation.